commands/management/add_mysql.go

```diff
@@ -163,6 +164,11 @@ func (cmd *addMySQLCommand) Run() (commands.Result, error) {
 		return nil, err
 	}
 
+	if cmd.CreateUser {
+		return nil, fmt.Errorf("Unrecognized option. To create a user, see" +
```

Comment:
I think it will look better with one space between `... see'https...`

Reply:
Sure. I miss that.